python/nammu/utils/__init__.py

```diff
@@ -192,34 +192,31 @@ def update_yaml_config(path_to_jar, yaml_path, path_to_config, verbose=False,
                     tmp = {}  # for the nested dics
                     for sub_key in jar_config[key]:
                         if sub_key in local_config[key]:
-                            logger.debug("%s: %s: %s --> Using local values.",
-                                         key,
-                                         sub_key,
-                                         jar_config[key][sub_key])
+                            logger.debug("{}: {}: {}} --> Using local values"
+                                         ".".format(key, sub_key,
```

Comment:
Double closing curly brackets.

Reply:
Wow, I would have thought these would have been flagged on loading Nammu. I'll fix them for this PR, but agree we shouldn't merge this PR at present.